src/main/java/by/dragonsurvivalteam/dragonsurvival/DragonSurvival.java

```diff
@@ -93,18 +93,28 @@ private void addPackFinders(AddPackFindersEvent event) {
             //resourcePacks.put(Component.literal("- Dragon North"), "resourcepacks/ds_north");
             //resourcePacks.put(Component.literal("- Dragon South"), "resourcepacks/ds_south");
             //resourcePacks.put(Component.literal("- Dragon West"), "resourcepacks/ds_west");
-            resourcePacks.put(Component.literal("- Old Magic Icons for DS"), "resourcepacks/ds_old_magic");
```

Comment:
Please put the “-” back in, this was added specifically to highlight resourcepacks from the list.